### PR TITLE
Update SoundVolumeView 2.23 -> 2.26

### PR DESCRIPTION
**Pull request by dragonmaus**

### CODE REVIEW

File: bucket/soundvolumeview.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     "checkver": "SoundVolumeView v([\\d.]+)",
-    "version": "2.23",
+    "version": "2.25",
     "license": "freeware",
     "description": "SoundVolumeView is a simple tool for Windows Vista/7/8/2008/10 that displays general information and current volume level for all active sound components on your system, and allows you to mute and unmute them instantly. SoundVolumeView also allows you to save a sound profile into a file, containing the current volume level and the mute/unmute state of all sound components, as well as the default sound devices, and then later, load the same file to restore exactly the same volume levels and settings.",
     "architecture": {
         "64bit": {
             "url": "https://www.nirsoft.net/utils/soundvolumeview-x64.zip",
-            "hash": "61b09a22830e4b809b48714217f08b7db7b9c0455eebca475c345f1dc7b8418b"
+            "hash": "4e839f0a9fdc6d1bf65709c2580a933ced5e16d1e0cdd487c439ca94bd271536"
```
Comment (kodybrown) [inline bucket/soundvolumeview.json:10]: what hash are you generating?

Comment (kodybrown) [reply bucket/soundvolumeview.json:10]: I think the zip may have already been updated..

Comment (kodybrown) [reply bucket/soundvolumeview.json:10]: I get:

```
> sha256.ps1 '.\soundvolumeview-x64.zip'
File:      .\soundvolumeview-x64 (1).zip
Algorithm: sha256
Hash:      3d43d8561b1bb2ebd6b9aba4468cf76d415ac1d8e0d211f30109d1f3717e67c6
```

Comment (dragonmaus) [reply bucket/soundvolumeview.json]: Yeah, looks like they snuck in another version update since I created the PR. 😅

Comment (kodybrown) [reply bucket/soundvolumeview.json]: And for x86:

```
sha256.ps1 '.\soundvolumeview.zip'
File:      .\soundvolumeview.zip
Algorithm: sha256
Hash:      8f9513a4e59c76d5093e179b5ebd5cbe93060721d076c9623188d65fc04c176a
```

Comment (dragonmaus) [reply bucket/soundvolumeview.json]: Added a commit to bump the version to 2.26.

**Comment by kodybrown** [comment]
Thanks!

**Comment by kodybrown** [comment]
Thank you for helping to keep this up to date!

Any thoughts on how to automate getting and updating all of NirSoft's utilities? I bet I don't even have half of all the utilities available. It's just a lot of work. Maybe I'll write a small C# or PowerShell script to scrape his main page, then output json files.. I'll take a look at it tonight (I hope).

**Comment by kodybrown** [comment]
Ohh.. He has PAD files. Nice! I'll check into automating the creation of scoop json files from those. That might be a good util for scoop itself.

**Comment by dragonmaus** [comment]
The PAD files look somewhat useful, but it seems you'd still have to do some scraping; it doesn't look like he includes 64-bit .zip files in those.

**Comment by dragonmaus** [comment]
Actually, you can probably just add -x64 to the end of the base filename of the given download URL, and if it doesn't exist assume that the app doesn't have a 64-bit version.

Other than that, you can extract just about everything in your JSON files from the PAD XML:
```
.homepage ← XML_DIZ_INFO.Web_Info.Application_URLs.Application_Info_URL
.checkver ← XML_DIZ_INFO.Program_Info.Program_Name + " v(\d\.\d\d)"
.version ← XML_DIZ_INFO.Program_Info.Program_Version
.license ← XML_DIZ_INFO.Program_Info.Program_Type (convert to lowercase)
.description ← XML_DIZ_INFO.Program_Descriptions.English.Char_Desc_2000
if 64-bit version exists:
    .architecture.64bit.url ← (autogenerated from .architecture.32bit.url)
    .architecture.64bit.hash ← (autogenerated from .architecture.64bit.url)
.architecture.32bit.url ← XML_DIZ_INFO.Web_Info.Download_URLs.Primary_Download_URL
.architecture.32bit.hash ← (autogenerated from .architecture.32bit.url)
if 64-bit version exists:
    .autoupdate.architecture.64bit.url ← .architecture.64bit.url
.autoupdate.architecture.32bit.url ← .architecture.32bit.url
.bin ← ""
.shortcuts[0][0] ← XML_DIZ_INFO.Program_Info.Program_Name + ".exe"
.shortcuts[0][1] ← "NirSoft\\" + XML_DIZ_INFO.Program_Info.Program_Name
```

**Comment by kodybrown** [comment]
Yeah, I noticed the missing x64's. I think you're right about them and the rest though. I'll tinker with it.. :)

**Comment by kodybrown** [comment]
I created a little script.. and checked it in.
There are still a few issues with the PAD files, but it added about 230 new utilities... :D

**Comment by kodybrown** [comment]
[aae4d76c2d7f22d62ccae96041326bdcdce90330](https://github.com/kodybrown/scoop-nirsoft/commit/aae4d76c2d7f22d62ccae96041326bdcdce90330)

**Comment by kodybrown** [comment]
It will be super easy to update now as well! I'm excited!

Thanks for the help. I used your code above and it saved some time. :)

**Comment by dragonmaus** [comment]
I'm working on a script too, and am pretty close to finishing. 😉
There are a few kinks to work out, but it's almost where I want it. I've not had any issues with the PAD files, so we'll see how they compare!

(I'm not used to working with PowerShell, so it's a bit rough at times. 😁)

**Comment by kodybrown** [comment]
Nice! I'd love to see what you come up with. And, I agree about PowerShell. Some things are just odd.

**Comment by dragonmaus** [comment]
One thing to note is that Powershell, by default, saves things using UTF-16 encoding, which produces invalid JSON files.

**Comment by dragonmaus** [comment]
*It is done.*

**Comment by dragonmaus** [comment]
Regardless of what you do, you definitely want to fix the UTF-16-encoded .json files, as they will not work with scoop.

**Comment by kodybrown** [comment]
Yeah, I saw that once I checked them in. I'll try passing them through jq after the conversion.

**Pull request merged**